### PR TITLE
Bug 1123300 - [Stingray] use querySelectorAll to query buttons

### PR DESCRIPTION
**Pull request by huchengtw-moz**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[huchengtw-moz John Hu (huchengtw-moz)](https://github.com/huchengtw-moz) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d5436c0f38f3)

File: tv_apps/tv_shared/js/scrollable.js [hunk 1]
```diff
@@ -57,7 +57,7 @@
 
     this.itemClassName = param.itemClassName;
     var items = Array.prototype.slice.call(
-                    document.getElementsByClassName(param.itemClassName));
+                    this.listElem.querySelectorAll('.' + param.itemClassName));
```
Comment (rexboy7) [inline tv_apps/tv_shared/js/scrollable.js]: does `this.listElem.getElementsByClassName(...)` works?

Comment (huchengtw-moz) [inline tv_apps/tv_shared/js/scrollable.js]: Yes, it also works. But I like querySelectorAll.....

**Comment by try-server-hook** [comment]
[huchengtw-moz John Hu (huchengtw-moz)](https://github.com/huchengtw-moz) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0dbe80e4b565)

**Pull request merged**